/**: 
  logger_node:
    ros__parameters:
      record_wheel_current: True
      record_wheel_voltage: True
      run_by_maestro: False
      use_sim_time: True
      type_of_encoder: 'JointState' #Either joint_states or topic
      left_wheel_encoder_velocity_index: 2
      right_wheel_encoder_velocity_index: 3
      cmd_msg_twist_stamped: True
      power_message_type: 'HuskyStatus'
      